README.md

```diff
@@ -223,7 +223,7 @@ Reference: https://www.link-to-info.io
 Tests:
   -
     Name: Name to describe our first test.
-    ResourceType: Resource.Type.Here
+    ResourceType: Resource.Type.Here  # Not needed after Panther version #TODO (the Panther version whose API doesnt need this field)
```

Comment:
Typo: `doesn't` (missing apostrophe)

Reply:
I think we cannot mark it deprecated because users *must* use it with the current/older Panther versions. Then after a target Panther version can they stop using it.

Reply:
Almost certainly this will be version 1.6 if you are also working on the `analysis-api` changes this sprint.

Reply:
Added version 1.6 in the comment and fixed the ci fail.